packages/eds-core-react/src/components/Popover/Popover.test.tsx

```diff
@@ -77,11 +78,14 @@ describe('Popover', () => {
 
     render(
       <TestPopover open onClose={handleOnClose}>
-        <div>some random content</div>
+        <Popover.Header>
```

Comment:
This test seems kinda redundant now as it only tests if the button was clicked. Suggest we just remove it.

Reply:
Just remove it :)